cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaMirrorMaker2ClusterTest.java

```diff
@@ -1249,7 +1249,7 @@ public void testGenerateDeploymentWithOldVersion() {
         assertThrows(InvalidResourceException.class, () -> {
             KafkaMirrorMaker2 resource = new KafkaMirrorMaker2Builder(this.resource)
                     .editSpec()
-                        .withVersion("2.3.1")  
+                        .withVersion("2.4.0")
```

Comment:
Should this be 2.5.0?

Reply:
Since the test is about `testGenerateDeploymentWithOldVersion`, I thought that the `n - 2` version should be tested. Have I misunderstood it?

Reply:
You are right. The 2.4.0 probably makes sense here 👍